### PR TITLE
Add GNU Nano spk

### PR DESCRIPTION
**Pull request by xeno010**
Add GNU Nano spk
Version 2.4.0 (stable)

### CODE REVIEW

File: cross/nano/PLIST [hunk 1]
```diff
@@ -0,0 +1,3 @@
+bin:bin/nano
+bin:bin/rnano
+shr:share
```
Comment (Diaoul) [inline cross/nano/PLIST]: What's in there that is required for nano to work?

File: spk/nano/Makefile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+SPK_NAME = nano
+SPK_VERS = 2.4.0
+SPK_REV = 1
+SPK_ICON = src/nano.png
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Nano is a text editor for Unix-like computing systems or operating environments using a command line interface.
```
Comment (Dr-Bean) [inline spk/nano/Makefile]: Can I suggest to be consistent with the naming? It's called `Nano` in the description, but the display names and changelog use `GNU Nano`. Perhaps simply using `nano` (lowercase, as per the website) would make the most sense?

Comment (Diaoul) [inline spk/nano/Makefile]: :+1:

**Comment by Dr-Bean** [comment]
Merged with https://github.com/SynoCommunity/spksrc/commit/27d2d9df813a5c9ab10570633a6061f2024c81ef, additional updates with https://github.com/SynoCommunity/spksrc/commit/9ff6b12f15bd2b9f6aaec71733d56e46c60c44c9.

Thanks for the addition, @xeno010 :)

**Pull request merged**